modules/backend/widgets/Search.php

```diff
@@ -1,4 +1,6 @@
-<?php namespace Backend\Widgets;
+<?php
```

Comment:
Leave this formatting the way it was please, that's the standard across all of October's files.